contracts/cw20-streams/src/error.rs

```diff
@@ -0,0 +1,38 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
```

Comment:
You will just need to `#[derive(PartialEq)]` here as well to allow the above check